Website/admin/Menus/ModuleActions/ModuleActions.ascx

```diff
@@ -1,94 +1,98 @@
-                supportsMove: <% = SupportsMove.ToString().ToLower() %>,
-                IsShared : <% = IsShared.ToString().ToLower() %>
-            });
+                    actionButton: "<% =actionButton.UniqueID %>",
+                    moduleId: moduleId,
+                    tabId: tabId,
+                    customActions: <% = CustomActionsJSON %>,
+                    adminActions: <% = AdminActionsJSON %>,
+                    panes: <% = Panes %>,
+                    customText: "<% = CustomText %>",
+                    adminText: "<% = AdminText %>",
+                    moveText: "<% = MoveText %>",
+                    topText: '<% = Localization.GetSafeJSString(LocalizeString("MoveTop.Action"))%>',
+                    upText: '<% = Localization.GetSafeJSString(LocalizeString("MoveUp.Action"))%>',
+                    downText: '<% = Localization.GetSafeJSString(LocalizeString("MoveDown.Action"))%>',
+                    bottomText: '<% = Localization.GetSafeJSString(LocalizeString("MoveBottom.Action"))%>',
+                    movePaneText: '<% = Localization.GetSafeJSString(LocalizeString("MoveToPane.Action"))%>',
+                    deleteText: '<%= Localization.GetSafeJSString("DeleteItem.Text", Localization.SharedResourceFile) %>',
+                    yesText: '<%= Localization.GetSafeJSString("Yes.Text", Localization.SharedResourceFile) %>',
+                    noText: '<%= Localization.GetSafeJSString("No.Text", Localization.SharedResourceFile) %>',
+                    confirmTitle: '<%= Localization.GetSafeJSString("Confirm.Text", Localization.SharedResourceFile) %>',
+                    rootFolder: '<%= Page.ResolveClientUrl("~/") %>',
+                    supportsMove: <% = SupportsMove.ToString().ToLower() %>,
+                    supportsQuickActions:supportsQuickActions,
+                    isShared : <% = IsShared.ToString().ToLower() %>,
+                    quickSettings: "#<%= quickSettings.ClientID %>"
+                },
+                {
+                    cancel: '<%=Localization.GetSafeJSString("Cancel", Localization.SharedResourceFile) %>',
+                    quickSettings: '<%=Localization.GetSafeJSString("QuickSettings", Localization.SharedResourceFile) %>',
+                    save: '<%=Localization.GetSafeJSString("Save", Localization.SharedResourceFile) %>'
+                }
+            );
+        }
 
-            $('#moduleActions-' + moduleId + ' .dnn_mact > li.actionMenuMove > ul').jScrollPane();
-            
-            $('#moduleActions-' + moduleId + ' .dnn_mact li').hoverIntent({
-                over: function() {
-                    // detect position
-                    var windowHeight = $(window).height();
-                    var windowScroll  = $(window).scrollTop();
-                    var thisTop = $(this).offset().top;
-                    var atViewPortTop = (thisTop - windowScroll) < windowHeight / 2;
+        $()
```

Comment:
Yup - not sure how that got there - it is obviously ignored as the code works :)